eve/workers/docker/Dockerfile

```diff
@@ -18,8 +18,16 @@ RUN yum install -y epel-release \
     python36-pip \
     git \
     docker-ce-cli-18.09.6 \
+    which \
     && adduser -u 1042 --home /home/eve eve --groups docker \
     && chown -R eve:eve /home/eve \
     && pip3 install buildbot-worker==${BUILDBOT_VERSION}
 
+
+RUN curl -LO https://github.com/oras-project/oras/releases/download/v0.12.0/oras_0.12.0_linux_amd64.tar.gz && \
+    mkdir -p oras-install/ && \
+    tar -zxf oras_0.12.0_*.tar.gz -C oras-install/ && \
```

Comment:
you should extract the version with a (dockerfile) buildarg, to make easier to update the version & avoid repetition:

```
ARG ORAS_VERSION=0.12.0
RUN curl -LO https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_amd64.tar.gz && \
    mkdir -p oras-install/ && \
    tar -zxf oras_${ORAS_VERSION}_*.tar.gz -C oras-install/ && \
    mv oras-install/oras /usr/local/bin/ && \
    rm -rf oras_${ORAS_VERSION}_*.tar.gz oras-install/

Reply:
will do.